### PR TITLE
Reduce healthcheck interval for agents.

### PR DESCRIPTION
**Pull request by mohsinenar**
Reduce healthcheck interval for agents.

### CODE REVIEW

File: src/ostorlab/runtimes/local/agent_runtime.py [hunk 1]
```diff
@@ -32,10 +32,10 @@
 HEALTHCHECK_HOST = "0.0.0.0"
 HEALTHCHECK_PORT = 5000
 SECOND = 1000000000
-HEALTHCHECK_RETRIES = 10
+HEALTHCHECK_RETRIES = 300
 HEALTHCHECK_TIMEOUT = 30 * SECOND
-HEALTHCHECK_START_PERIOD = 2 * SECOND
-HEALTHCHECK_INTERVAL = 30 * SECOND
+HEALTHCHECK_START_PERIOD = 1 * SECOND
+HEALTHCHECK_INTERVAL = 1 * SECOND
```
Comment (deadly-panda) [inline src/ostorlab/runtimes/local/agent_runtime.py:38]: Should probably apply the same value to the lite local.

File: src/ostorlab/agent/agent.py [hunk 2]
```diff
@@ -91,7 +91,11 @@ def __init__(
         self._agent_settings = agent_settings
         self._control_message: Optional[agent_message.Message] = None
         self.name = agent_definition.name
-        self.in_selectors = agent_definition.in_selectors
+        self.in_selectors = (
```
Comment (3asm) [inline src/ostorlab/agent/agent.py:94]: Please update you PR branch. These are not related to your changes.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: src/ostorlab/runtimes/local/agent_runtime.py (hunk 1 above)
Comment (3asm) [reply src/ostorlab/runtimes/local/agent_runtime.py:38]: +1.

**Review by 3asm — CHANGES_REQUESTED** [review]

File: src/ostorlab/runtimes/local/agent_runtime.py (hunk 1 above)
Comment (mohsinenar) [reply src/ostorlab/runtimes/local/agent_runtime.py:38]: @deadly-panda @3asm I did!

**Review by deadly-panda — DISMISSED** [review]

**Review by oussamaessaji — APPROVED** [review]

**Review by 3asm — APPROVED** [review]
